cmd/main.go

```diff
@@ -37,7 +37,8 @@ func main() {
 
 	openPrometheusAndPprof(cfg.Debug.ListenAddr)
 
-	log.Infow("bsc mev-sentry start", "configPath", *configPath, "config", cfg)
+	hidedCfg, _ := cfg.HidePrivateKey()
+	log.Infow("bsc mev-sentry start", "configPath", *configPath, "config", hidedCfg)
```

Comment:
I dont think it is necessary to print the whole setting in the log. ( Do not see similar logic in other repo like BSC, opBNB)